### PR TITLE
sub and sub elements style added in typography scss file

### PR DESCRIPTION
**Pull request by akshayvinchurkar1**
sub and sub elements style added in typography scss file

### CODE REVIEW

**Comment by 0crat** [comment]
@filfreire please, pay attention to this pull request

**Comment by 0crat** [comment]
Job `gh:yegor256/tacit#112` is in scope.

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta name="keywords" content="css, html5, css framework"/>
     <meta name="description" content="CSS Framework for Dummies"/>
-    <link rel="stylesheet" href="tacit.min.css"/>
+    <link rel="stylesheet" href="dist/tacit-css-0.0.0.min.css"/>
```
Comment (filfreire) [inline index.html]: @akshayvinchurkar1 this isn't right, we shouldn't point to `0.0.0` version here.

File: scss/_typography.scss [hunk 2]
```diff
@@ -65,3 +65,19 @@ ins,
...
+  vertical-align: baseline;
+}
+
+sup {
+  top: -0.5em;
+}
+
+sub {
+  bottom: -0.25em;
+}
```
Comment (filfreire) [inline scss/_typography.scss]: @akshayvinchurkar1 could you add the missing new-line here?

**Comment by filfreire** [comment]
@akshayvinchurkar1 please have a look at the review comments 🙂

**Pull request closed without merging**